index.bs

```diff
@@ -10316,18 +10318,14 @@ an exception when called as a function.
         [=overload resolution algorithm=].
     1.  Let |R| be the result of performing the actions listed in the description of
         |constructor| with |values| as the argument values.
-    1.  Return the result of [=converted to an ECMAScript value|converting=]
+    1.  Rethrow any [=exception=] thrown in the previous step.
```

Comment:
Don't [simple exceptions](https://heycam.github.io/webidl/#dfn-simple-exception) englobe all of the ES errors that can be thrown at runtime?